### PR TITLE
Add Y3D space

### PR DESCRIPTION
**Pull request by guanchao71**
**Add space for Y3D**  
  
Etherscan: https://etherscan.io/address/0xc7fd9ae2cf8542d71186877e21107e1f3a0b55ef  
Website: https://y3d.finance/y3d/ycrv/
Twitter: https://twitter.com/Y3dScam
Debank: https://debank.com/projects/y3d

### CODE REVIEW

File: spaces/y3d/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.name {
```
Comment (e-asia) [inline spaces/y3d/skin.scss:1]: The css selector should similar to the space key.
```suggestion
.y3d {
```

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/y3d/skin.scss (hunk 1 above)
Comment (guanchao71) [reply spaces/y3d/skin.scss]: I have completed the modification

**Comment by lychees** [comment]
@e-asia Kindly ping.

File: spaces/y3d/index.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "key": "Y3D",
```
Comment (e-asia) [inline spaces/y3d/index.json:2]: The key of your space must also be the folder name and skin class name.
```suggestion
  "key": "y3d",
```

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/y3d/index.json (hunk 2 above)
Comment (guanchao71) [reply spaces/y3d/index.json]: I am too careless

**Review by e-asia — APPROVED** [review]

**Pull request merged**